GUI.lua

```diff
@@ -116,6 +117,14 @@ function ArenaStats:CreateGUI()
     arenaTypeDropdown:SetValue(filters.arenaType)
     asGui.f:AddChild(arenaTypeDropdown)
 
+    local nameFilter = AceGUI:Create("EditBox")
+    nameFilter:SetLabel("Filter By Name")
```

Comment:
Do we need to localize this string?

Reply:
Yes, you're definitely correct. Need that string to be localized